client/www/js/controllers/routeFilterController.js

```diff
@@ -26,7 +26,16 @@ angular.module('amblr.routeFilter', [])
 
     $scope.setRoute = function (routeID) {
       $scope.filteredRoute = routeID;
-      POIs.setRouteFilter(routeID);
+      if (routeID) {
```

Comment:
@zelifus Why do you need the conditional? Is it possible to do a filter without receiving an id or for the id given to not exist in the available routes?